package.json

```diff
@@ -62,7 +62,9 @@
     "posttest": "yarn lint",
     "prepack": "yarn build && oclif manifest && oclif readme",
     "prepare": "yarn build",
-    "test": "mocha --forbid-only \"test/**/*.test.ts\"",
+    "test": "mocha \"test/**/*.test.ts\" --exit",
+    "test:ci": "mocha --forbid-only \"test/**/*.test.ts\" --exit",
```

Comment:
`--exit` is currently needed because msw leaves some timeouts (or similar?) hanging.

I broke out `test:ci` so I could use `only` locally. We _do_ want to forbid it on ci, though.